internal/pkg/template/app.go

```diff
@@ -24,12 +27,22 @@ type AppNestedStackOpts struct {
 	PolicyOutputs   []string
 }
 
+// AppHealthCheckOpts holds configuration for the container healthcheck.
+type AppHealthCheckOpts struct {
+	Command     []string
```

Comment:
I notice we already have the right format for command in manifest. Can we make it to be a string so that we don't need quoteall and stringifyslice?

Reply:
Unfortunately, `Command` in CFN is a list of strings that's why I kept it like that: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-healthcheck.html#cfn-ecs-taskdefinition-healthcheck-command

This struct is meant to match how healthcheck is defined in cfn. For example, `Interval` is an int whereas in the manifest it's a `time.Duration`.
I wonder if i should have just use the sdk's cloudformation struct instead: https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#HealthCheck. That seems to make more sense 🤔

Reply:
Yeah I think we might want to use the defined struct.

Reply:
Done!

Reply:
Honestly, I'm not a fan of bring an sdk dependency to the `template` package. I wonder if I should create a separate `cf/template` pkg and move all the `ParseApp` methods there.

Reply:
Why are you concerned about this? Is it because of the possiblity of cyclic dependencies, or just a desire to minimize the spread of external dependencies of our packages?

Reply:
The latter.
We have a generic `template` pkg that can read any sort of Go template that's located under our "templates/" directory (like manifests, cf, custom resources). So it felt a bit weird that this package would require the SDK. Maybe a better option down the line is to separate out reading cf templates to its own package, built on top of this one, where it makes sense to require the SDK.